embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testcases/NetworkRequestApiTest.kt

```diff
@@ -263,12 +264,20 @@ internal class NetworkRequestApiTest {
                     assertEquals(expectedRequest.bytesReceived.toString(), this.attributes["http.response.body.size"])
                     assertEquals(null, this.attributes["error.type"])
                     assertEquals(null, this.attributes["error.message"])
+                    val statusCode = expectedRequest.responseCode
+                    val expectedStatus = if ( statusCode != null && statusCode >= 200 && statusCode < 400) {
```

Comment:
```suggestion
                    val expectedStatus = if (statusCode != null && statusCode >= 200 && statusCode < 400) {
```

Reply:
Haha can't believe this wasn't caught by lint